content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,103 @@ description: Upgrading an ambient mode installation with Helm.
 
-    {{< text syntax=bash snip_id=manual_crd_upgrade >}}
-    $ kubectl apply -f manifests/charts/base/crds
+    {{< text syntax=bash snip_id=list_revisions >}}
+    $ kubectl get mutatingwebhookconfigurations -l 'istio.io/rev,!istio.io/tag' -L istio\.io/rev
+    $ # Store your revision and new revision in variables:
+    $ export REVISION=istio-1-22-1
+    $ export OLD_REVISION=istio-1-21-2
     {{< /text >}}
 
-1. Upgrade the Istio base chart:
+## Upgrade the Kubernetes custom resource definitions ({{< gloss >}}CRDs{{</ gloss >}})
 
-    {{< text syntax=bash snip_id=upgrade_base >}}
-    $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
+    {{< text bash >}}
+    $ kubectl apply -f manifests/charts/base/crds
     {{< /text >}}
 
-### Upgrade the Istio discovery Component
+## Install the new Control Plane
 
-Istiod is the control plane component that manages and configures the proxies to route traffic within an ambient mesh.
+Istiod is the control plane component that manages and configures the proxies to route traffic within an ambient mesh. The following command will install a new instance of this control plane alongside the old, but will not introduce any new proxies, or take over control for existing proxies. If you have previously customized your istiod installation, you can reuse the `values.yaml` file from previous upgrades or installs to keep your control planes consistent.
 
 {{< text syntax=bash snip_id=upgrade_istiod >}}
-$ helm upgrade istiod istio/istiod -n istio-system
+$ helm install istiod-"$REVISION" istio/istiod -n istio-system --set revision="$REVISION" --set profile=ambient --wait
 {{< /text >}}
 
-### Upgrade the ztunnel component
+## Upgrade the ztunnel DaemonSet
 
-The ztunnel DaemonSet is the node proxy component.
-
-{{< warning >}}
-As ambient mode is not yet Stable, the following statement is not a compatibility guarantee, and is subject to change, or removal. Prior to reaching Stable status, this component and/or the control plane may receive breaking changes that prevent compatibility between minor versions.
-{{< /warning >}}
-
-The ztunnel at version 1.x is generally compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version.
+The {{< gloss >}}ztunnel{{< /gloss >}} DaemonSet is the node proxy component. The ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version. If you have previously customized your ztunnel installation, you can reuse the `values.yaml` file from previous upgrades or installs to keep your {{< gloss >}}data plane{{< /gloss >}} consistent.
```

Comment:
```suggestion
The ztunnel `DaemonSet` is the node proxy component. ztunnel at version 1.x is compatible with the control plane at version 1.x+1 and 1.x. This means the control plane must be upgraded before ztunnel, as long as their version difference is within one minor version. 

If you have previously customized your ztunnel installation, you can reuse the `values.yaml` file from previous upgrades or installs to keep your data plane consistent.
```

Reply:
This gloss was requested by a previous reviewer, so I'm hesitant to remove it.

Reply:
I will have done that because you should gloss on first mention.